src/content/apis/foods/NutritionAnalysisAPI.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Edamam
+description: Do nutrition analysis by Copy/paste any food recipe and learn its nutrition details in under a second
+categories:
+  - foods
+features:
+  https: true
+url: https://developer.edamam.com/edamam-nutrition-api
+logo: ![small-logo](https://user-images.githubusercontent.com/67019423/193456793-834cee23-1d15-4cfb-8eed-04bdaf733bdb.png)
```

Comment:
The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.